google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudType.java

```diff
@@ -97,7 +97,8 @@ public void connect(@NotNull ConnectionCallback<AppEngineDeploymentConfiguration
         return;
       }
 
-      if (!CloudSdkService.getInstance().isValidCloudSdk()) {
+      CloudSdkValidator sdkValidator = CloudSdkValidator.getSdkValidator();
+      if (!sdkValidator.isValidCloudSdk()) {
```

Comment:
nit: you can probably just do `CloudSdkValidator.getInstance().isValidCloudSdk()` in the if clause instead of extracting a variable.

Reply:
Agree.